components/calendar/src/iso.rs

```diff
@@ -14,10 +14,10 @@ pub struct Iso;
 
 /// A 1-indexed representation of an ISO day
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
-pub struct IsoDay(u8);
+pub struct IsoDay(pub u8);
```

Comment:
issue: These shouldn't be made public: they're private to maintain the invariants on the inner field. Feel free to add From impls to integers if needed.